src/blocks/folder-blocks/minimap/index.tsx

```diff
@@ -1,6 +1,6 @@
 import { useMemo } from "react";
 // @ts-ignore
-import { Tree } from "./Tree.jsx";
```

Comment:
For some reason, this was needed to run it locally. But the vite imports are all messed up anyway, so 🤷‍♀️